doc/examples/example_html.details.html

```diff
@@ -86,10 +86,10 @@ <h1>GCC Code Coverage Report</h1>
       <a href="example_html.details.example.cpp.9597a7a3397b8e3a48116e2a3afb4154.html">example.cpp</a>
     </th>
     <td>
-      <meter class="coverage-medium" min="0" max="100" value="85.7" title="85.7%">85.7</meter>
+      <meter class="coverage-medium" min="0" max="100" value="87.5" title="87.5%">87.5</meter>
     </td>
-    <td class="CoverValue line-coverage coverage-medium">85.7%</td>
-    <td class="CoverValue line-coverage coverage-medium">6 / 7</td>
+    <td class="CoverValue line-coverage coverage-medium">87.5%</td>
+    <td class="CoverValue line-coverage coverage-medium">7 / 8</td>
```

Comment:
Why did the coverage report change if the test stayed the same? This is worrying.

Reply:
That wasn't intended. I think, that was because of a different compiler version. I'll revert this.

Reply:
This test must only be used with `gcc-5` or `gcc-6`, the other compilers change the data.